src/browser/components/MainPage.jsx

```diff
@@ -602,13 +602,20 @@ export default class MainPage extends React.Component {
   };
 
   showDownloadCompleteNotification = async (event, item) => {
-    const serverName = process.platform === 'win32' ? item.serverInfo.origin : 'Download Complete';
+    const regexResult = (/https?:\/\/(.+).mattermost.com\/?$/).exec(item.serverInfo.origin);
```

Comment:
There wasn't a servername parameter exposed, all I got was the server url.

Reply:
but 3 lines below (608) we are accessing `serverName`:
```js
       serverName = item.serverInfo.serverName; 
```
if it isn't there, that line would fail, right?

Reply:
Sorry, maybe this is not the best name for the variable, but this is actually the server URL and if the regex doesn't match the URL (ie, its not a mattermost url) the serverName would actually be the url.